cmd/earthly/flags.go

```diff
@@ -323,6 +323,13 @@ func (app *earthlyApp) buildFlags() []cli.Flag {
 			Required:    false,
 			Destination: &app.orgName,
 		},
+		&cli.StringFlag{
+			Name:        "project",
+			EnvVars:     []string{"EARTHLY_PROJECT"},
+			Usage:       wrap("The name of the organization the satellite belongs to. ", "Required when using --satellite and user is a member of multiple organizations."),
+			Required:    false,
+			Destination: &app.projectName,
+		},
```

Comment:
Should this flag be hidden? Is there a good use-case for the user to specify this themselves?

Reply:
Good point. Updated.